src/main/java/lotto/LottoCustomer.java

```diff
@@ -11,13 +13,14 @@
 
     public static void main(String[] args) {
         int price = InputView.requestPrice();
+        List<LottoNums> lottoNumsList = InputView.requestManualNumsList().stream().map(LottoNums::of).collect(Collectors.toList());
 
         LottoSeller lottoSeller = new LottoSeller();
-        LottoPaper lottoPaper = lottoSeller.sell(price);
+        LottoPaper lottoPaper = lottoSeller.sell(price, lottoNumsList);
 
         OutputView.printLottos(lottoPaper.getLottoLines());
 
-        LottoNums lottoNums = LottoNums.of(InputView.requestNums());
+        LottoNums lottoNums = LottoNums.of(InputView.requestWinNums());
```

Comment:
여기도 당첨 번호라는걸 알 수 있는 변수명으로 지정하면 어떨까요?
```suggestion
        LottoNums winNums = LottoNums.of(InputView.requestWinNums());
```